src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -64,29 +50,302 @@
 public class TemplateController implements HandlerExceptionResolver {
 
     protected static Logger logger = Logger.getLogger(TemplateController.class);
+
     @Autowired
     ServletContext servletContext;
+
+    @Resource(name = "dataManager")
+    private DataManager dataManager;
+    /*
+    @Resource(name = "jsonManager")
+    private JsonManager jsonManager;
+    */
     @Resource(name = "resourceManager")
     private ResourceManager resourceManager;
-    @Resource(name = "fileParserManager")
-    private FileParserManager fileParserManager;
+
+
+
+    /*
     @Resource(name = "netcdfFileManager")
     private NetcdfFileManager netcdfFileManager;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)